openedx/course-discovery
# feat: migrate DjangoChoices to native django choices (models.TextChoices)

[PROD-3521](https://2u-internal.atlassian.net/browse/PROD-3521)
----------------


This PR migrates dj-choices `Django Choices` to native django choices `models.TextChoices`. 


Django docs regarding native django choices:
  - https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types

Django-choices contains documentation for the migration [Migrating to native Django Choices — Django-Choices 2.0.0 documentation](https://django-choices.readthedocs.io/en/latest/migrating.html) .



Testing:
- Check from discovery django admin from each of choices field working correctly.
- Go to publisher Course Edit Page and try to change the value of each choice field.
- Change the choices values through `api/v1/` endpoints
